transports/jetty/src/main/java/org/mule/transport/servlet/jetty/JettyMessageProcessTemplate.java

```diff
@@ -102,13 +104,24 @@ public void sendResponseToClient(MuleEvent muleEvent) throws MuleException
         }
     }
 
+    private Map<String, String> getThrottlingHeaders() {
+        Map<String, String> throttlingHeaders = httpThrottlingHeadersMapBuilder.build();
+
+        for ( String headerName : extraHeaders.keySet() )
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Iterate over the "entrySet" instead of the "keySet". ([squid:S2864](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS2864))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7625&issues=AWiOv--GotIXnmsU1tFw&open=AWiOv--GotIXnmsU1tFw)

[ISSUE_KEY]: AWiOv--GotIXnmsU1tFw